stencil.config.ts

```diff
@@ -111,6 +112,9 @@ export const create: () => Config = () => ({
         stylelint({
           configFile: ".stylelintrc-postcss.json",
           fix: true
+        }),
+        cssnano({
+          preset: "default"
```

Comment:
this default comes with SVGO which I think is already built into icons. This might cause an issue? Not sure, just want to point it out.

Reply:
Thanks for bring this up. The icon assets (path data) [shouldn't get processed by `postcss`](https://github.com/stencil-community/stencil-postcss/tree/master#valid-file-extensions), so I _think_ we're good here.